README.md

```diff
@@ -53,6 +53,18 @@ appraisal_matrix(activesupport: "6.1") do
 end
 ```
 
+#### Block arguments
+If you would like to setup conditional logic based off of the versions of the gems in the matrix, you can pass a block with arguments to `appraisal_matrix`.
+
+```ruby
+appraisal_matrix(activesupport: "6.1") do |activesupport:, sidekiq:|
+  # activesupport <Gem::Version>
+  if activesupport < "7"
```

Comment:
One more ⛏️
```suggestion
  if activesupport < Gem::Version.new("7")
```

Reply:
This is only necessary on Ruby 3.0, I like how clean it is in Ruby 3.1+ to just leave it as a string